src/domains/chart/components/lib-charts/dygraph-chart.tsx

```diff
@@ -831,21 +831,29 @@ export const DygraphChart = ({
       // todo support state.tmp.dygraph_force_zoom
       const forceDateWindow = [viewAfter, viewBefore]
 
+      const dygraphOptionInEffect = dygraphInstance.current.getOption("dateWindow")
```

Comment:
does `getOption("dateWindow")` give different results than `xAxisRange()` below? It's not clear why different method is used

Reply:
Didn't realise this was the same value. Fixed